test/end2end_test.go

```diff
@@ -206,7 +206,7 @@ func (s *testServer) StreamingOutputCall(args *testpb.StreamingOutputCallRequest
 	return nil
 }
 
-func (s *testServer) StreamingInputCall(stream testpb.TestService_StreamingInputCallServer) error {
+func (s *testServer) StreamingInputCall(stream testpb.TestServiceStreamingInputCallServer) error {
```

Comment:
Type `TestService_StreamingInputCallServer` is defined by generated code, so this line cannot be changed.